<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>581</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>datetime</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>sex</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>age</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>menu</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
